features/backend/product_attributes.feature

```diff
@@ -28,99 +28,74 @@ Feature: Product attributes
         When I press "Create"
         Then I should still be on the product attribute creation page
         And I should see "Please enter attribute name"
-        And I should see "Please enter attribute presentation"
+        And I should see "Please enter attribute code"
 
-    Scenario: Creating new attribute
+    @javascript
+    Scenario: Accessing attribute creation form
+        Given I am on the product attribute index page
+        When I click "Create"
+        And I choose "checkbox" attribute type
+        Then I should be on the product attribute creation page for "checkbox" type
+        And I should see select "Type" with "Checkbox" option selected
+        And I should not be able to edit "Type" select
+
+    Scenario: Accessing attribute creation form without type specified
         Given I am on the product attribute creation page
-        When I fill in "Internal name" with "Book author"
-        And I fill in "Presentation" with "Author"
+        And I should see select "Type" with "Text" option selected
+        And I should not be able to edit "Type" select
+
+    Scenario: Creating new attribute
+        Given I am on the product attribute creation page with type "text"
+        When I fill in "Name" with "ISBN number"
+        And I fill in "Code" with "isbn"
         And I press "Create"
         Then I should be on the product attribute index page
         And I should see "Attribute has been successfully created."
         And I should see 3 attributes in the list
-        And I should see attribute with name "Book author" in that list
+        And I should see attribute with name "ISBN number" in that list
+
+    Scenario: Creating new attribute with configuration
+        Given I am on the product attribute creation page with type "date"
+        When I fill in the following:
+            | Name   | Released at  |
+            | Code   | release_date |
+            | Format | d/m/Y        |
+        And I press "Create"
+        Then I should be on the product attribute index page
+        And I should see "Attribute has been successfully created."
+        And I should see 3 attributes in the list
+        And I should see attribute with name "Released at" in that list
+
+     Scenario: Creating new attribute with validation options
```

Comment:
I guess we should have at least one scenario for validating if the validation options works (the validation is really applied).